pip-constraint.txt

```diff
@@ -0,0 +1,4 @@
+# Used to test compatibility with oldest supported dependencies
+appdirs==1.4.3
```

Comment:
shouldn't we pin all dependencies here?

Reply:
I pinned only a subset which did not cause any troubles. We will do the rest in follow-ups as we may have to make other changes. This change does not change what we ship.

Reply:
😢 I'd personally rather have it fixed now if we have stuff that's causing issues

Reply:
I'd recommend making it one commit per dependency though 👍